mail/api_test.py

```diff
@@ -576,3 +589,146 @@ def test_send_to_course_team_error(self, raise_for_status, mock_post):
         assert response.raise_for_status.called is raise_for_status
         assert response.status_code == HTTP_400_BAD_REQUEST
         assert response.json() == {}
+
+
+class AutomaticEmailTests(MockedESTestCase):
+    """Tests regarding automatic emails"""
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.program_enrollment_unsent = ProgramEnrollmentFactory.create()
+        cls.program_enrollment_sent = ProgramEnrollmentFactory.create()
+        with mute_signals(post_save):
+            cls.percolate_query = PercolateQuery.objects.create(
+                query="query",
+                original_query="original",
+            )
+            cls.percolate_query_disabled = PercolateQuery.objects.create(
+                query="other query",
+                original_query="other original",
+            )
+        cls.automatic_email = AutomaticEmail.objects.create(
+            query=cls.percolate_query,
+            enabled=True,
+            email_subject='subject',
+            email_body='body',
+            sender_name='Sender name',
+        )
+        cls.automatic_email_disabled = AutomaticEmail.objects.create(
+            query=None,
```

Comment:
Did you mean to set `cls.percolate_query_disabled` as the `query` here? If so, you might want to take a second look at the test logic since it's working without the intended conditions